multi_host_migration/tests/migration_multi_host_with_reboot.py

```diff
@@ -84,7 +84,7 @@ def migration_scenario(self, worker=None):
                 min_t = int(params.get("min_random_timeout"))
                 max_t = int(params.get("max_random_timeout"))
```

Comment:
see a bug here.  it should be params["max/min_random_timeout"], covert  None object to int will raise TypeError here